docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
-
-* GitHub
-	* Github is a software that runs locally on your computer and your files and their history are stored on your computer. Having a centrally located place to upload files and make changes and download changes from others enables you to collaborate more easily with other developers. 
+- Git
+	Git is a version control system that lets you track who made changes to what files
+	and at what time. A folder of code that is tracked together by git is called a
+	repository. Git runs locally on your computer and keeps track of the changes you
+	make and allows you to see all the past changes you and others have made. It also
+	makes collaboration easier by making it easy to coordinate changes by multiple
+	people. You can access the git via command line (terminal) or a desktop app that
+	has a GUI, such as GitKraken (strongly recommended!). 
 
+- GitHub
+	- Github is a website that provides hosting of git repositories and adds some very
+	helpful collaboration tools as well. It provides a centrally located, always online,
+	access point for git repositories.
 	
-* Python 
-	* Python is a high-level, general-purpose, interpreted computer programming language. It is widely applied in many fields to build solutions for web applications, software development, data analysis and conducting AI techniques. Python has two major versions: 2x was released in 2000 and the latest version 3x was released in 2008. It’s recommended to use the latest version 3 in projects.
+- Python 
+	- Python is a high-level, general-purpose, interpreted computer programming
+	language. It is widely applied in many fields and it is a relatively easy
+	programming language to learn. Python is particularly good at making it easy to
+	leverage code libraries written by other programmers and there is a very large,
+	robust python open source community.
 
 ## Opening up a terminal 
-*For Mac devices:*
-Click the Launchpad icon in the Dock, type Terminal in the search field, then click Terminal. In the Finder, open the /Applications/Utilities folder, then double-click Terminal.
-
-*For Windows devices:*
-Press the Start taskbar button. Select All apps on the Start menu. Scroll down the Start menu to the Windows Terminal app or Command Prompt app (Windows Terminal is a little nicer if you have it but not required) shown directly below. Then click Windows Terminal or Command Prompt there to open it.
+- **On Mac:**
+Click the Launchpad icon in the Dock, type Terminal in the search field, then click
+Terminal. In the Finder, open the /Applications/Utilities folder, then double-click
+Terminal.
 
+- **On Windows:**
+Press the Start taskbar button. Select All apps on the Start menu. Scroll down the
```

Comment:
The format of the taskbar unfortunately varies a lot from one version of windows to another -- Windows 11 doesn't have "All apps" 😵 . All of them support just starting to type and having apps come up in search, however. Consider rewording to `Press the Start taskbar button and start typing to search for either the 'Terminal' app or, if it's not installed, the 'Command prompt' app.`